e2e-test/src/tests/cleanUp.test.js

```diff
@@ -0,0 +1,106 @@
+import {
+  vBtn,
+  vListTile,
+  vIcon,
+} from 'jest-puppeteer-vuetify';
+import { CLIENT_URL } from '../util';
+
+
+describe('relabel dataset', () => {
+  it('make dataset ready for analysis with 2 clicks', async () => {
```

Comment:
There might be a better way to name this

Reply:
How about `mark as group`? That's the relabeling action that is being done in this test.